examples/advanced_operations/add_responsive_search_ad_full.py

```diff
@@ -25,12 +25,12 @@
 import argparse
 import sys
 import uuid
+
 import requests
 
 from google.ads.googleads.client import GoogleAdsClient
 from google.ads.googleads.errors import GoogleAdsException
 
-
 # Keywords from user.
 _KEYWORD_TEXT_EXACT_1 = "example of exact match"
```

Comment:
Could these just be "KEYWORD_TEXT_EXACT", "KEYWORD_TEXT_PHRASE", "KEYWORD_TEXT_BROAD"? I don't see why the _1s are beneficial (they make sense for the other constants but not these)